.travis.yml

```diff
@@ -1,12 +1,22 @@
 language: rust
-sudo: required
+sudo: false
+
 rust:
   - nightly
   - beta
   - stable
-after_success: |
-  cargo doc && \
-  echo '<meta http-equiv=refresh content=0;url=rss/index.html>' > target/doc/index.html && \
-  sudo pip install ghp-import && \
-  ghp-import -n target/doc && \
-  git push -qf https://${TOKEN}@github.com/${TRAVIS_REPO_SLUG}.git gh-pages
+
+before_script:
+  - pip install 'travis-cargo<0.2' --user && export PATH=$HOME/.local/bin:$PATH
```

Comment:
Using docs.rs would probably be a good idea. Maybe we could use both, use travis-cargo for documentation on the master branch and use docs.rs for the current released version.